framework/src/modules/chain/blocks/process.js

```diff
@@ -92,7 +92,10 @@ class BlocksProcess {
 		const context = {
 			blockTimestamp: timestamp,
 			blockHeight: lastBlock.height + 1,
-			blockVersion: blockVersion.currentBlockVersion,
+			blockVersion: blockVersion.getBlockVersion(
+				lastBlock.height + 1,
+				this.exceptions
```

Comment:
you shouldn't need to pass the `exceptions` here because it should only generate the latest block version